frameworks/PHP/swoole/swoole-server-noasync.php

```diff
@@ -9,7 +9,7 @@
 ]);
 
 $pdo = new PDO("mysql:host=tfb-database;dbname=hello_world", "benchmarkdbuser", "benchmarkdbpass", [
-    PDO::ATTR_PERSISTENT => true
+        PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION
```

Comment:
I don't think you'll get any additional error messages.
I displayed the Mysql error stats on the second last run travis-ci, and for `Swoole-no-async`, you had nothing (see https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/598272528#L3623)

Reply:
By default PDO use PDO::ERRMODE_SILENT.
https://www.php.net/manual/en/pdo.error-handling.php

And perhaps the error is for reusing the same db connection in the pool.

Reply:
And some errors:
https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/598272528#L3607-L3608

Reply:
> And some errors:
> https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/598272528#L3607-L3608

Good point, you have your answer:
too many simultaneous requests on the same PDO connection

Reply:
But I need to add the PDO Exception, or fail silently.

Reply:
You can do both:
- Display errors
- Limit the chances of having one by creating the pdo connection for each worker

Reply:
I know, I used so in Workerman and nginx. But it's not posible using the original anonymous functions.